src/main/java/seedu/duke/notebooks/Section.java

```diff
@@ -43,6 +43,15 @@ public void addPage(Page p) {
         pageArrayList.add(p);
     }
 
+    public void getPage(int pageIndex) {
+        try {
+            Page page = pageArrayList.get(pageIndex);
+            page.printPage();
+        } catch (IndexOutOfBoundsException | NullPointerException e) {
+            System.out.println("page <" + (pageIndex + 1) + "> doesn't exist");
+        }
+    }
+
```

Comment:
do we still need this, now that we're using page title?

Reply:
True

Reply:
We can just mark as `@Deprecated`